pkg/kubelet/kubelet_pods_test.go

```diff
@@ -2595,6 +2595,85 @@ func Test_generateAPIPodStatus(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
I expected both of these to test that message and reason are preserved from previous state

Reply:
Done!